ext/pdo_pgsql/pdo_pgsql.c

```diff
@@ -22,9 +22,13 @@
 #include "php_ini.h"
 #include "ext/standard/info.h"
 #include "pdo/php_pdo.h"
+#include "pdo/php_pdo_int.h"
 #include "pdo/php_pdo_driver.h"
 #include "php_pdo_pgsql.h"
 #include "php_pdo_pgsql_int.h"
+#include "pdo_pgsql_arginfo.h"
+
+zend_class_entry *PdoPgsql_ce;
```

Comment:
Same "why is this a global" question here. Actually each driver seems to have their own global CE, so the question could be repeated for each of them :P